### PR TITLE
scripts: update according to arguments unification

### PR DESCRIPTION
**Pull request by hachreak**
* Updates script according to last change in invenio-accounts cli.
  (addresses invenio-accounts#56)

Signed-off-by: Leonardo Rossi <leonardo.r@cern.ch>

### CODE REVIEW

**Comment by jirikuncar** [comment]
@hachreak `s/accoounts/accounts `

File: scripts/populate-instance.sh [hunk 1]
```diff
@@ -106,7 +106,7 @@ ${INVENIO_WEB_INSTANCE} db create
 
 # sphinxdoc-create-user-account-begin
 ${INVENIO_WEB_INSTANCE} users create \
-       --email ${INVENIO_USER_EMAIL} \
+       ${INVENIO_USER_EMAIL} \
```
Comment (tiborsimko) [inline scripts/populate-instance.sh]: Maybe pass first the optional arguments and second the mandatory arguments, as is the usual style?
(`invenio3 users create --foo1 bar1 --foo2 bar2 xyzzy zyxxy`)

Comment (jirikuncar) [inline scripts/populate-instance.sh]: IMHO it's just a habit. I prefer arguments first and then options.

Comment (tiborsimko) [inline scripts/populate-instance.sh]: It's a widespread convention in Unix, see e.g. any system command manual page `man ls` and friends, the advertised usage synopses are `ls [options] ... <file> ...`, or `kill [options] <pid>`, etc... Kinda everywhere.

**Pull request merged**